ts/components/conversation/message/message-content/MessageContextMenu.tsx

```diff
@@ -360,18 +348,14 @@ export const MessageContextMenu = (props: Props) => {
         </StyledEmojiPanelContainer>
       )}
       <SessionContextMenuContainer>
-        <Menu
-          id={contextMenuId}
-          onShown={onContextMenuShown}
-          onHidden={onContextMenuHidden}
-          animation={animation.fade}
-        >
+        <Menu id={contextMenuId} onVisibilityChange={onVisibilityChange} animation="fade">
           {enableReactions && (
             // eslint-disable-next-line @typescript-eslint/no-misused-promises
             <MessageReactBar action={onEmojiClick} additionalAction={onShowEmoji} />
           )}
-          <SaveAttachment messageId={messageId} />
-
+          {attachments?.length ? (
+            <Item onClick={saveAttachment}>{window.i18n('downloadAttachment')}</Item>
```

Comment:
Why not `&&` here instead? Then you don't need to return `null`?

Reply:
that'd be the "pitfall" of https://react.dev/learn/conditional-rendering#conditionally-including-jsx